lib/features/tips_card_switcher.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:math';
+
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:udsp59/features/tips_card.dart';
+
+class TipsCardSwitcher extends StatefulWidget {
+  final List<String> tips;
+
+  const TipsCardSwitcher({
+    super.key,
+    required this.tips,
+  });
+
+  @override
+  State<TipsCardSwitcher> createState() => _TipsCardSwitcherState();
+}
+
+class _TipsCardSwitcherState extends State<TipsCardSwitcher> {
+  var state1 = 0;
+  var state2 = 1;
+
+  @override
+  void initState() {
+    super.initState();
+    debugPrint(widget.tips.toString());
+    if (widget.tips.isNotEmpty) {
+      state1 = Random().nextInt(widget.tips.length);
+      if (widget.tips.length >= 2) {
+        state2 = state1 == widget.tips.length - 1 ? 0 : state1 + 1;
+      } else {
+        state2 = state1;
+      }
+    }
+    debugPrint('state1: $state1');
+    debugPrint('state2: $state2');
```

Comment:
what are state1 and state2 ? 

instead of using random directly inside the UI you should use a state manager like Riverpod